test/users/engin/compiler/innerRecord.future

```diff
@@ -0,0 +1,8 @@
+bug: nil dereference referencing outer class' fields
+
+NOTES
+
+- Where outer,inner == class,class; same thing happens, which I would understand
+  due to memory allocation differences btw records vs classes
+
+- Where outer == record, nothing works(it is a separate future)
```

Comment:
It'd be good to name the other future rather than forcing the reader to search for it.  You might add more futures to this directory, or rearrange the directory structure in the future, and so the connection that seems obvious now might be lost and not as easily recovered